restconf_update_ipaddress/updateip.py

```diff
@@ -149,13 +165,29 @@ def get_ip_info():
 
 
 def main():
-    global do_input
-
     """
     Simple main method calling our function.
     """
+
+    parser = ArgumentParser(
+        prog=sys.argv[0], description='RESTCONF interface management tool')
+    parser.add_argument('--hostname', '-a', type=str,
+                        help='sandbox hostname or IP address',
+                        default='ios-xe-mgmt.cisco.com')
+    parser.add_argument('--username', '-u', type=str,
+                        help='sandbox username', default='developer')
+    parser.add_argument('--port', '-P', type=int,
+                        help='sandbox web port', default=443)
+    args = parser.parse_args()
+
+    password = getpass()
```

Comment:
Do you have a preference for a variable name?  Also, it seems less secure to me to take the password from an environment variable.

Reply:
Maybe DEVNET_RESTCONF_PASSWORD.  Reading from the environment is slightly more secure than the CLI since only administrators could grab that data.  But admins could also get to the memory if they wanted.

Reply:
done